File: install_help/install_r_packages.py
import rpy2.robjects as robjects
import os

lib = 'r_lib'

if not os.path.exists(lib):
    os.mkdir(lib)

depends = [
    "readr",
    ]
    # "data.table",
    # "futile.logger",
    # "future",
    # "future.apply",
    # "ggplot2",
    # "lubridate",
    # "patchwork",
    # "progressr",
    # "purrr",
    # "R.utils", 
    # "Rcpp",
    # "runner",
    # "scales",
    # "truncnorm",
    # "BH",
    # "RcppEigen",
    # "RcppParallel",
    # "StanHeaders",
    # "rstan",
    # "rstantools",
    # "knitr"]


def install():

    #for package in depends:
    #    robjects.r(f'install.packages("{package}", dependencies=TRUE, lib="{lib}", INSTALL_opts = "--no-multiarch")')
    #robjects.r(f'install.packages("EpiNow2", dependencies=TRUE, lib="{lib}", repos = "https:/epiforecasts.r-universe.dev", INSTALL_opts = "--no-multiarch")')
    robjects.r(f'install.packages("EpiNow2", dependencies=TRUE, lib="{lib}", INSTALL_opts = "--no-multiarch")')

install()

File: install_help/install_r_packages.R
lib = 'r_lib'

local({r <- getOption("repos")
      r["CRAN"] <- "https://cran.csiro.au/"
      options(repos=r)})

list_data <- list("readr","data.table","futile.logger","future","future.apply","ggplot2","lubridate",
"patchwork","progressr","purrr","R.utils", "Rcpp","runner","scales","truncnorm","BH","RcppEigen",
"RcppParallel","StanHeaders","rstan","rstantools","knitr")

# install.packages("EpiNow2", repos = "https:/epiforecasts.r-universe.dev", dependencies=TRUE, lib=lib, INSTALL_opts = "--no-multiarch")

for (val in list_data)
{
    install.packages(val, dependencies=TRUE, lib=lib, INSTALL_opts = "--no-multiarch")
}

